### DISCUSSION TITLE
Dịch từ "instance"

### DISCUSSION THREAD

**Discussion by rootonchair** [d2l-ai/d2l-vi#2993] [Terminology]
`Instance` là viết tắt của từ `cloud instance`, một khu vực trên mây nhằm cung cấp dịch vụ điện toán cho người dùng, thường là một máy ảo. Theo mọi người thì mình có nên dịch từ này không?

@lkhphuc @minhduc0711 @duythanhvn @cuongvng @davidnvq @trGiang99

**Comment by trgiangdo** [comment]
Cái này mình nghĩ là nên dịch, nếu giữ nguyên `instance` sợ bị nhầm lẫn với `thực thể` trong hướng đối tượng.

**Reply by lkhphuc** [reply]
Dịch là `Một thực thể máy ảo a1.4xlarge ...` cũng được mà nhỉ.

**Reply by cuongvng** [reply]
Hơi cồng kềnh ạ. Em nghĩ `máy ảo` là ổn rồi, vì instance chỗ này cũng không có special meaning lắm.

**Reply by rootonchair** [reply]
ok, vậy mình chốt là dùng `máy ảo` nhé
